server/requirements-lint.txt

```diff
@@ -1,11 +1,10 @@
 flake8==4.0.1
 flake8-bugbear==22.4.25
 flake8-docstrings==1.6.0
-flake8-import-order==0.18.1
 flake8-quotes==3.3.1
-flake8-commas==2.1.0
```

Comment:
this will be enforced by black:

https://pypi.org/project/flake8-commas/
Project description
No Maintenance Intended
Note: [Black](https://pypi.org/project/black/), the uncompromising Python code formatter, or [add-trailing-comma](https://github.com/asottile/add-trailing-comma) can do all this comma insertion automatically. We recommend you use one of those tools instead.